packages/twenty-server/src/engine/metadata-modules/agent/dtos/agent.dto.ts

```diff
@@ -17,6 +23,14 @@ export class AgentDTO {
   @Field()
   name: string;
```

Comment:
There's something unclear about our distinction between name and label, I think we should implement exactly the same system we have for objects or fields, where you type the Label and the name is automatically computed ; unless if you're in advanced mode then you can edit the name